### PR TITLE
Include core massaction in reception list to allow using doMassActions hook

### PR DESCRIPTION
**Pull request by vincentjdc**
# New - *call doMassActions hook in reception list*
The hook doMassActions is now called when doing a mass action in the reception list

### CODE REVIEW

File: htdocs/reception/list.php [hunk 1]
```diff
@@ -166,6 +169,10 @@
 }
 
 if (empty($reshook)) {
+	$uploaddir = $conf->reception->dir_output.'/temp/massgeneration/'.$user->id;
+	$objectclass = 'Reception';
+	include DOL_DOCUMENT_ROOT.'/core/actions_massactions.inc.php';
```
Comment (eldy) [inline htdocs/reception/list.php:174]: There is this test into this file
	if (empty($diroutputmassaction)) {
		dol_print_error(null, 'include of actions_massactions.inc.php is done but var $diroutputmassaction was not defined');
		exit;
	}

So why having remove the

diroutputmassaction = $conf->reception->dir_output.'/temp/massgeneration/'.$user->id;

Comment (vincentjdc) [reply htdocs/reception/list.php:174]: You're right. I did not noticed that `$diroutputmassaction` was used in actions_massactions.inc.php. I tested it with custom massaction (that was not `builddoc`) so I did not have that error.

`$uploaddir` could be the same as `$diroutputmassaction` ?

Comment (eldy) [reply htdocs/reception/list.php:174]: $uploaddir is the root of module. So
$uploaddir = $conf->reception->dir_output;
$diroutputmassaction is a temporary sub-dir of this directory.

**Comment by eldy** [comment]
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.

**Pull request closed without merging**